### DISCUSSION TITLE
How to jump to next groups?

### DISCUSSION THREAD

**Discussion by jorgemxm** [folke/flash.nvim#342] [Q&A]
Hi, when using the "f" command to find by character, and when the jump_labels are enabled, what's the command to Jump to the next  group of labels? There are `a` letters that have blue color instead of green. And that Do NOT have a label to jump to. 
What does that secondary highlight color means? for instance:

<img width="600" alt="image" src="https://github.com/folke/flash.nvim/assets/5948446/0fb8dcc9-003d-42c9-be00-8817a3bebac2">

Example settings:
```lua
{
    "folke/flash.nvim",
    event = "VeryLazy",
    vscode = true,
    opts = {
      modes = {
        char = {
          jump_labels = true, -- 
        },
      }
    },
}
```

I wasn't able to find answers similar to this topic, nor in the plugin  help. Thanks in advance

**Comment by JoeDaBu** [comment]
I'd like to know this too

**Comment by allpan3** [comment]
It means those characters are too far out of the reach such that they cannot be mapped to an available label letter.

**Comment by DaniBedz** [comment]
It feels like you should be able to hit `]` or similar to move the window of available labels down if it's too far ahead. Would love this feature.
